addons/account/views/res_partner_bank_views.xml

```diff
@@ -57,8 +57,7 @@
             <field name="model">res.partner.bank</field>
             <field name="inherit_id" ref="base.view_partner_bank_search"/>
             <field name="arch" type="xml">
-                <xpath expr="//search" position="replace">
-                    <search string="Bank Accounts">
+                <xpath expr="//search" position="inside">
                         <filter string="Trusted" name="trusted" domain="[('allow_out_payment', '=', True)]"/>
```

Comment:
possibly a `<separator/>` should be added above this line

Reply:
sorry. above the line `<filter string="Trusted" name="trusted" domain="[('allow_out_payment', '=', True)]"/>`, so there would be a line between Trusted/Untrusted/To validate and the filter/fields from the original search view.

Reply:
sure, it's a small thing, so I don't think it really matters. But since it's not visible at the moment, feels a bit useless. And if someone adds something in the future they can always add the separator then